vendor/phantomjs/render.js

```diff
@@ -55,7 +55,8 @@
 
         var rootScope = body.injector().get('$rootScope');
         if (!rootScope) {return false;}
-        return rootScope.panelsRendered;
+        var panelsToLoad = window.angular.element('div.panel').length;
```

Comment:
But 'panel-drop-zone' can be exist in whole dashboard. When rendering whole dashboard with all panels panelsToLoad=window.angular.element('div.panel').length return more than needed so render.js can stack here.